service/src/test/java/bio/terra/workspace/service/workspace/AzureCloneWorkspaceTest.java

```diff
@@ -125,4 +147,121 @@ void cloneAzureWorkspaceWithContainer() {
             .size(),
         1);
   }
+
+  @Autowired private SamService samService;
+  @Autowired private LandingZoneApiDispatch landingZoneApiDispatch;
+  @Autowired private AzureConfiguration azureConfig;
+
+  @Test
+  void testK8s() throws ApiException {
+    var azureProfile =
+        new AzureProfile(
+            "fad90753-2022-4456-9b0a-c7e5b934e408",
+            "c5f8eca3-f512-48cb-b01f-f19f1af9014c",
+            AzureEnvironment.AZURE);
+    var containerServiceManager =
+        ContainerServiceManager.authenticate(getManagedAppCredentials(), azureProfile);
+
+    //    var bearerToken = new BearerToken(samService.getWsmServiceAccountToken());
+    //    UUID landingZoneId = UUID.fromString("75f4d9ce-7aaf-496a-b39e-aa420a6ac26c");
+    //    var clusterResource = landingZoneApiDispatch.getSharedKubernetesCluster(bearerToken,
+    // landingZoneId).orElseThrow(() -> new RuntimeException("No shared cluster found"));
+    final String mrgName = "test-cf3afe3d-579d-4f97-b7fe-c07d5a6bc967";
+    final String aksClusterName = "lzd3f6108103145738f161ef0";
+    var rawKubeConfig =
+        containerServiceManager
+            .kubernetesClusters()
+            .manager()
+            .serviceClient()
+            .getManagedClusters()
+            .listClusterUserCredentials(mrgName, aksClusterName)
+            .kubeconfigs()
+            .stream()
+            .findFirst()
+            .orElseThrow(() -> new RuntimeException("No kubeconfig found"));
+    var kubeConfig =
+        KubeConfig.loadKubeConfig(
+            new InputStreamReader(new ByteArrayInputStream(rawKubeConfig.value())));
+    var userToken = kubeConfig.getCredentials().get("token");
+
+    ApiClient client =
+        Config.fromToken(kubeConfig.getServer(), userToken)
+            .setSslCaCert(
+                new ByteArrayInputStream(
+                    Base64.getDecoder()
+                        .decode(kubeConfig.getCertificateAuthorityData().getBytes())));
+    CoreV1Api api = new CoreV1Api(client);
+    var shellCommand =
+        String.join(
+            " && ",
+            List.of(
+                "apt update",
+                "apt -y install curl",
+                "curl -sL https://aka.ms/InstallAzureCLIDeb | bash",
+                "az login --federated-token \"$(cat $AZURE_FEDERATED_TOKEN_FILE)\" --service-principal -u $AZURE_CLIENT_ID -t $AZURE_TENANT_ID --allow-no-subscriptions",
+                "psql \"host=${DB_SERVER_NAME}.postgres.database.azure.com port=5432 dbname=postgres user=${ADMIN_DB_USER_NAME} password=$(az account get-access-token --query accessToken -otsv) sslmode=require\" --command \"CREATE DATABASE ${NEW_DB_NAME};\"",
```

Comment:
Do we want all of this in the test?

Reply:
Perhaps this was all just a way for you to test locally and you didn't intend to check in the file?

Reply:
sorry, this is junk